contributor-ladder.md

```diff
@@ -0,0 +1,222 @@
+    * Experience as a Reviewer for at least [TODO: Number] months
+    * Demonstrates a broad knowledge of the project across multiple areas
+    * Is able to exercise judgement for the good of the project, independant of their employer, friends, or team
+    * Mentors other contributors
+    * Can commit to spending at least [TODO: Number] hours per month working on the project
+* Additional privileges:
+    * Approve PRs to any area of the project
+    * Represent the project in public as a Maintainer
+    * Communicate with the CNCF on behalf of the project
+    * Have a vote in Maintainer decision-making meetings
+
+Process of becoming a maintainer:
+<!-- TODO: this process will vary widely across projects, both because of project code structure, and because of project governance.  For example, in some projects the Steering Committee approves new Maintainers.  What's below is just an example from a simple project in which the maintainers are also the project leaders, and which uses Github OWNERS files. -->
+1. Any current Maintainer may nominate a current Reviewer to become a new Maintainer, by opening a PR against the root of the [TODO: main repository name] adding the nominee as an Approver in the OWNERS file.
+2. The nominee will add a comment to the PR testifying that they agree to all requirements of becoming a Maintainer.
+3. A majority of the current Maintainers must then approve the PR.
+
+
+<!-- ### ADDITIONAL MAINTAINER ROLES
+Some projects have additional, specifically defined maintainer roles because of leadership positions that don't fit within the general maintainer template above, including ones that have special requirements.  In addition to spelling out those requirements, defining and publishing additional maintainer roles can be a way to recruit maintainers in those specific areas, especially non-code maintainers.  Here's examples of special maintainer roles which have been defined by a variety of projects.  As all of these roles are highly dependant on exact project organization, tooling, etc., these roles are not templatable.
+
+* Subproject Maintainer: Owns a distict subproject or repository of the main project.  Responsible for everything there.  In federation projects, subproject maintainers might be the primary maintainter type.  [Example]()
+* Documentation Maintainer: for your Docs leads.  Would include specific documentation targets and experience, and maintaining publication schedules as a requirement.  [Example]()
+* Localizations Maintainers: owns a particular localization, like Japanese or Brazillian Portuguese, across documentation, messages, and websites.  Responsible for making sure those get updated.  [Example]()
+* Program Manager: Responsible for timelines and processes within the project, such as bug triage, review timelines, etc.  [Example]()
+* Release Manager: owns the release process, either always, cyclically, or per-release.  Responsible for getting releases out on time.  [Example]()
```

Comment:
Is there a template for the Release Manager role, yet?

Reply:
Nope, if you know of an example, then please give me a link!

Reply:
No, we don't have that yet.

Reply:
I'm just linking to the Kubernetes Release Team docs, then.